play/PlayerRules/Basic Rules.txt

```diff
@@ -521,6 +565,51 @@ Hunkering costs one action, and you cannot fire your weapons. Imagine
 curling up in a ball, to pose the absolute smallest target possible. Its a 
 free action to un-hunker
 
+Stealth Attacks:
+If you are stealthed (undetected by an opponent) when you initiate an attack
+action, you incur bonuses depending on the circumstances of the attack.
+	Ranged weapon stealth-attacks (including explosive but not any other thrown 
+weapons) have normal damage with -1 miss chance, if 'adequate' aiming time is 
+given (DM's discretion: how rushed you are, or how skilled you are at stealth, etc).

```

Comment:
You're viewing an older commit, the crit effect is already in.

The "Hit the Dirt" isn't supposed to be a choice because it represents natural instincts of self-preservation and/or any sort of military experience. Unless the fantasy world gives the player the choice to eliminate fear and primal instincts of their character, they'd probably dive to the ground when they get shot at. Unlike how the combat is played right now, where every character could stand around in a circle and take turns ghatting each other if everyone so chose. Also I'd, personally, prefer to remove it from the superbranch altogether than slow down turns more with a check.

Also, if a  player gets caught out in the open and the opponent has 3 actions of 6-bullet automatic fire to spray on them, this will help prevent the game from being overly punishing of single mistakes or single low roll.  And vice-versa, the enemies won't be walked over as easily, but I don't think that is necessarily a good thing.

And this all ties into suppression as a concept: has more to do with realism, self-preservation, and fear of death than anything else. But I'm trying to make it game-playable too.